.changeset/gold-shirts-yell.md

```diff
@@ -0,0 +1,6 @@
+---
+'@cloudfour/patterns': minor
```

Comment:
This is a major change now, the name of the exported function changed, and now it requires calling `init()`. Could you copy-paste one of the code examples from the docs into the changeset briefly showing the new usage?

Reply:
https://github.com/cloudfour/cloudfour.com-patterns/pull/1882/commits/7eeac0072f15509ec5b509a340d3019cfc80157c and https://github.com/cloudfour/cloudfour.com-patterns/pull/1882/commits/6921adc1fe6861aed2b6c075a0ebef469194ef51